css/languages.css

```diff
@@ -44,16 +59,20 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
+  list-style: none;
+  gap: 8px;
 }
 
 @media screen and (min-width: 768px) {
   .language-main-container {
     display: grid;
-    grid-template-columns: 327px 327px 327px;
-    grid-template-rows: 352px;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: 1fr;
     column-gap: 24px;
-    justify-content: center;
-    padding-top: 52px;
-    padding-bottom: 88px;
+    height: 352px;
+    margin: 0 135px;
+    background-color: #fff;
+    margin-top: 52px;
+    margin-bottom: 84px;
   }
 }
```

Comment:
As request by the previous Code Reviewer(CR) please consider removing the horizontal scrollbar that is occurring because of the cards under about me section. 

## Elaboration:--

Currently, this scrollbar appears on the screens between (768px and 1050px).

## Solution and Reasoning:-

The scroll bar appears when the width of the object is greater than the width of the screen. This means if you consider verifying or reducing width in the above-mentioned range it will be solved. But remember to make it like the design so it looks professional and attractive also.